src/commands/services/export.ts

```diff
@@ -0,0 +1,136 @@
+  ALL = "ALL",
+  LOCAL = "LOCAL",
+  NATIONAL = "NATIONAL"
+}
+
+export default class VisibleServicesExport extends Command {
+  public static description = "Export visible services";
+
+  public static flags = {
+    day: flags.string({
+      description:
+        "filter services from specified day (Europe/Rome timezone, required format yyyy-MM-dd, ie 2020-05-25)",
+      required: false
+    }),
+    extended: flags.boolean({ char: "x", description: "show extra columns" }),
+    scope: flags.string({
+      char: "s",
+      description: "The service scope metadata",
+      required: false,
+      default: ScopeFilter.ALL,
+      options: [ScopeFilter.ALL, ScopeFilter.LOCAL, ScopeFilter.NATIONAL]
+    })
+  };
+
+  // tslint:disable-next-line: cognitive-complexity
+  public async run(): Promise<void> {
+    const { flags: parsedFlags } = this.parse(VisibleServicesExport);
+
+    const day = fromNullable(parsedFlags.day).map(_ =>
+      DateTime.fromFormat(`${_} Europe/Rome`, "yyyy-MM-dd z")
+    );
+
+    if (isSome(day) && !day.value.isValid) {
+      this.error("day is not valid");
+      return;
+    }
+    try {
+      cli.action.start("Querying services...");
+      const services = await getServices(day);
+      cli.action.stop();
+
+      if (isNone(services)) {
+        this.error("No result");
+        return;
+      }
+
+      cli.action.start("Filtering visible services...");
+      // get all visible services
+      const visible = services.value.filter(x => {
+        if (x.isVisible === false) {
+          return false;
+        }
+        if (parsedFlags.scope === ScopeFilter.LOCAL) {
+          return x.serviceMetadata?.scope === ServiceScopeEnum.LOCAL;
+        }
+        if (parsedFlags.scope === ScopeFilter.NATIONAL) {
+          // Services without metadata are considered NATIONAL scoped
+          return (
+            x.serviceMetadata?.scope === ServiceScopeEnum.NATIONAL ||
+            x.serviceMetadata === undefined
+          );
+        }
+        return true;
+      });
+      cli.action.stop();
+
+      cli.action.start("Remap services...");
+      const serviceMapper = getServiceMapper(parsedFlags.extended);
+      const organizationMap = visible.reduce((prev, _) => {
+        if (prev[_.organizationFiscalCode]) {
+          return {
+            ...prev,
+            [_.organizationFiscalCode]: {
+              ...prev[_.organizationFiscalCode],
+              s: [...prev[_.organizationFiscalCode].s, serviceMapper(_)]
+            }
+          };
+        }
+        return {
+          ...prev,
+          [_.organizationFiscalCode]: {
+            o: _.organizationName,
+            fc: _.organizationFiscalCode,
+            s: [serviceMapper(_)]
+          }
+        };
+      }, {} as Record<string, ServicesExport>);
+      cli.action.stop();
+      // tslint:disable-next-line: no-console
+      console.log(
+        JSON.stringify(
+          Object.values(organizationMap).sort((a, b) => {
```

Comment:
```suggestion
          Object.values(organizationMap).sort((a, b) => a.o.toLowerCase().localeCompare(b.o.toLowerCase())
```
Could it be a simplification?

Reply:
Fixed in c36e77c